postgres/changelog.d/16199.changed

```diff
@@ -0,0 +1 @@
+INCIDENT-23437 Always resolve hostname instead of returning agent hostname regardless or dbm or generic tags enabled or not
```

Comment:
Should we add a more meaningful changelog entry here you think? Given that its a breaking change

Reply:
That makes sense. I'll try to think of something but if you've got another example of a great breaking changelog entry, it would be nice.